app/models/nta_upload.rb

```diff
@@ -295,10 +295,10 @@ def make_country_geos(row_data)
                                display_name_short: row_data['subregn'],
                                geotype: 'region2',
                                parents: geo_region.id })
-    income_grp = (row_data['incgrp'] || row_data['incgrp2015']).sub(/.income$/i, '').titlecase
+    income_grp = (row_data['incgrp'] || row_data['incgrp2015']).sub(/\s*countries.*$/i, '').titlecase
     geo_incgrp = Geography.get_or_new_nta({ handle: income_grp.gsub(/\W+/, '_') },
-                                          { display_name: "#{income_grp} Income",
-                                            display_name_short: "#{income_grp} Income",
+                                          { display_name: "#{income_grp} Countries",
```

Comment:
Does this add "Countries" anywhere they didn't.

Reply:
No, that's their change.